doc/sphinx/user/methods/basic-equations/2d-models.md

```diff
@@ -12,15 +12,15 @@ As a consequence, we ought to really think of these two-dimensional models as th
 If one adopts this point of view, the Stokes equations {math:numref}`eq:stokes-1`-{math:numref}`eq:stokes-2` naturally simplify in a way that allows us to reduce the $3+1$ equations to only $2+1$, but it makes clear that the correct description of the compressible strain rate is still $\varepsilon(\mathbf u) - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1$, rather than using a factor of $\frac{1}{2}$ for the second term.
 (A derivation of why the compressible strain rate tensor has this form can be found in {cite:t}`schubert:etal:2001`, sec. 6.5.)
 
-It is interesting to realize that this compressible strain rate indeed requires a $3\times 3$ tensor. While under the assumptions above, we have
+Because we think of these 2d models as just a slice through an infinite cylinder along the $z$-axis, we should still see what happens if we express the strain rate as a $3\times 3$ tensor. Under the assumptions above, we have
 ```{math}
 \begin{aligned}
   \varepsilon(\mathbf u) = \begin{pmatrix} \tfrac{\partial u_x}{\partial x} & \tfrac 12 \tfrac{\partial u_x}{\partial y} + \tfrac 12 \tfrac{\partial u_y}{\partial x} & 0 \\
     \tfrac 12 \tfrac{\partial u_x}{\partial y} + \tfrac 12 \tfrac{\partial u_y}{\partial x} & \tfrac{\partial u_y}{\partial y} & 0 \\
     0 & 0 & 0 \end{pmatrix}
 \end{aligned}
 ```
-with the expected zeros in the last row and column, the full compressible strain rate tensor reads
+with the expected zeros in the last row and column: the last row contains derivatives of $u_z=0$, whereas the last column contains $z$-derivatives of all velocities (which are also zero under our assumption). Because the strain rate tensor is zero all $z$-related entries, our assumption about what 2d models represent is often calls "[plane strain](https://en.wikipedia.org/wiki/Infinitesimal_strain_theory#Plane_strain)" in mechanics, indicating that nonzero strain components are all in the $x$-$y$ plane. Interestingly, though, for compressible materials, the full compressible strain rate tensor then reads
```

Comment:
```suggestion
with the expected zeros in the last row and column: the last row contains derivatives of $u_z=0$, whereas the last column contains $z$-derivatives of all velocities (which are also zero under our assumption). Because the strain rate tensor is zero for all $z$-related entries, our assumption about what 2d models represent is often called "[plane strain](https://en.wikipedia.org/wiki/Infinitesimal_strain_theory#Plane_strain)" in mechanics, indicating that nonzero strain components are all in the $x$-$y$ plane. Interestingly, though, for compressible materials, the full compressible strain rate tensor then reads
```